### DISCUSSION TITLE
Requirements for a project to be added to a collection?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [community/community#23979] [New to GitHub]
<p>GitHub allows people to suggest their own edits to GitHub collections. The <a href="https://github.com/github/explore/blob/287c155c08f842919ebcae2ec8b5ba03b4989f81/CONTRIBUTING.md" rel="noopener nofollow ugc">CONTRIBUTING</a> file covers most important points, but I cannot find information on the requirements for a project to be added to a collection.</p>
<p>Should a project-candidate be well-known? Or have at least some fixed number of users? Or have mentions somewhere?</p>
<p>To be precise, I would like to know about the requirements for a project to be added to <code>software-in-science</code>.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>The only criteria I see is that it’s popular enough that someone felt it merited inclusion. As part of the process of adding a project, you will be asked to confirm that you are not affiliated with it - this keeps the lists from becoming self-promotion and ensures that only projects found to be valuable by the community will be included.</p>

**Reply by Brad212** [reply]
Do you think my account: brad212 has any problem as I was told I cannot be added for a github project.
